resources/definitions/prusa_i3_mk2.def.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "prusa_i3_mk2",
+    "version": 2,
+    "name": "Prusa i3 Mk2",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Apsu",
+        "manufacturer": "Prusa Research",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "icon": "icon_ultimaker2",
+        "platform": "prusai3_platform.stl",
+        "has_materials": true
+    },
+
+    "overrides": {
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 250 },
+        "machine_height": { "default_value": 200 },
+        "machine_depth": { "default_value": 210 },
+        "machine_center_is_zero": { "default_value": false },
+        "material_diameter": { "default_value": 1.75 },
+        "material_bed_temperature": { "default_value": 55 },
+        "machine_nozzle_size": { "default_value": 0.4 },
+        "layer_height": { "default_value": 0.1 },
+        "layer_height_0": { "default_value": 0.15 },
+        "retraction_amount": { "default_value": 0.8 },
+        "retraction_speed": { "default_value": 35 },
+        "retraction_retract_speed": { "default_value": 35 },
+        "retraction_prime_speed": { "default_value": 35 },
+        "adhesion_type": { "default_value": "skirt" },
+        "machine_nozzle_heat_up_speed": { "default_value": 2 },
+        "machine_nozzle_cool_down_speed": { "default_value": 2 },
+        "machine_head_with_fans_polygon": { "default_value": [[0,0],[0,0],[0,0],[0,0]] },
+        "gantry_height": { "default_value": 0 },
```

Comment:
Slight clarification: The gantry_height is used when printing in "one at a time" mode to prevent the gantry from colliding with previous prints. So if you set gantry_height to anything lower than the build volume height, the build volume height will be shrunk vertically. By setting the gantry_height to 0 there will be no remaining build volume. That is why by default fdmprinter.json uses a very high value (99999 if I am not mistaken) to impose no such limitation.